product_docs/docs/epas/16/epas_rel_notes/epas16_5_0_rel_notes.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: EDB Postgres Advanced Server 16.5.0 release notes
+navTitle: "Version 16.5.0 "
+---
+
+Released: 14 Nov 2024
+
+EDB Postgres Advanced Server 16.5.0  includes the following enhancements and bug fixes:
+
+| Type           | Description                                                                                                                                         | Addresses&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; |
+|----------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------|
+| Upstream merge | Merged with community PostgreSQL 15.9. See the [PostgreSQL 15.9 Release Notes](https://www.postgresql.org/docs/release/15.9/) for more information. |                                                                                                     |
```

Comment:
I think this should be [16.5](https://www.postgresql.org/docs/16/release-16-5.html).
```suggestion
| Upstream merge | Merged with community PostgreSQL 16.5. See the [PostgreSQL 16.5 Release Notes](https://www.postgresql.org/docs/release/16.5/) for more information. |                                                                                                     |
```